cmd/template/app/flag.go

```diff
@@ -6,18 +6,20 @@ import (
 )
 
 const (
-	flagCatalog       = "catalog"
-	flagCluster       = "cluster"
-	flagName          = "name"
-	flagNamespace     = "namespace"
-	flagUserConfigMap = "user-configmap"
-	flagUserSecret    = "user-secret"
-	flagVersion       = "version"
+	flagCatalog           = "catalog"
+	flagCluster           = "cluster"
+	flagDefaultingEnabled = "defaulting-enabled"
```

Comment:
See the docs PR but the defaulting is only enabled for these releases. As we didn't want to impact existing app CRs.

- AWS >= v14.0.0
- Azure >= v13.1.0
- KVM >= v13.1.0

If this flag is set to false then all fields will be set.